feeds/wlan-ap/opensync/files/bin/wlan_ap_redirector.sh

```diff
@@ -1,11 +1,30 @@
 #!/bin/sh
 
-if [ $# -ne 1 ] ; then
-	echo "Usage: $0 <redirector address>" >&2
-	exit 1
+AP_PRIVATE_KEY_FILE="/usr/opensync/certs/client_dec.key"
+AP_CERTIFICATE_FILE="/usr/opensync/certs/client.pem"
+AP_DEVICE_ID_FILE="/usr/opensync/certs/deviceid.txt"
```

Comment:
The file /usr/opensync/certs/deviceid.txt does not exist on my AP.  
What is the procedure to create this information.

Reply:
@brennerm can you please update the PR to reflect the correct Device ID file and also while you're here update the digicert API URL to production instance.